### ISSUE TITLE
Unable to update_data

### ISSUE THREAD

**Issue by cmodyssey** [tchellomello/python-ring-doorbell#248]
Hi,

I am trying to use this for the first time and I am getting a problem with getting started on this module.

I sure do have an active subscription, we have a number of ring devices active in our household.

I was also prompted for the 2FA password that was sent by SMS.

Of the two times I ran the script, the first time it needed the 2FA password and the second time, it had cached all the credentials.

Below is the result I get.  I have also attached the script I have been trying to get to work.

Does anyone know what could be the problem please?

Thanks.

`

C:\Temp>test.py
homeauto script started
Username: ******@******
Password:
2FA code: ******
Traceback (most recent call last):
  File "C:\Temp\test.py", line 73, in <module>
    main()
  File "C:\Temp\test.py", line 66, in main
    ring.update_data()
  File "C:\Users\Richard\AppData\Local\Programs\Python\Python37\lib\site-packages\ring_doorbell\__init__.py", line 58, in update_data
    self.update_groups()
  File "C:\Users\Richard\AppData\Local\Programs\Python\Python37\lib\site-packages\ring_doorbell\__init__.py", line 99, in update_groups
    for group in data["device_groups"]:
TypeError: 'NoneType' object is not iterable

C:\Temp>test.py
homeauto script started
Traceback (most recent call last):
  File "C:\Temp\test.py", line 73, in <module>
    main()
  File "C:\Temp\test.py", line 66, in main
    ring.update_data()
  File "C:\Users\Richard\AppData\Local\Programs\Python\Python37\lib\site-packages\ring_doorbell\__init__.py", line 58, in update_data
    self.update_groups()
  File "C:\Users\Richard\AppData\Local\Programs\Python\Python37\lib\site-packages\ring_doorbell\__init__.py", line 99, in update_groups
    for group in data["device_groups"]:
TypeError: 'NoneType' object is not iterable`

[test.zip](https://github.com/tchellomello/python-ring-doorbell/files/6414029/test.zip)

**Comment by Gamerios** [comment]
same here:
root@Server:~/ringdownload# python3 test.py
Traceback (most recent call last):
  File "test.py", line 49, in <module>
    main()
  File "test.py", line 34, in main
    ring.update_data()
  File "/root/ringdownload/ring_doorbell/__init__.py", line 58, in update_data
    self.update_groups()
  File "/root/ringdownload/ring_doorbell/__init__.py", line 99, in update_groups
    for group in data["device_groups"]:
TypeError: 'NoneType' object is not iterable
root@Server:~/ringdownload#

**Comment by cmodyssey** [comment]
Hi @Gamerios 

Thanks for taking a look.

Do you know how to get it working though?

Thanks.

**Comment by Gamerios** [comment]
**yes Need use 0.6.2 becaue 0.7.0 has that bug**

**Comment by Gamerios** [comment]
**thanks how you found that if i can ask**

**Comment by cmodyssey** [comment]
Hi, I saw a comment only the other day saying it's worth trying the pull requests and when I viewed the 2 here and saw the symptom sounded like mine, so gave it a go and voila!

**Comment by Gamerios** [comment]
**at you download work ??**

**Comment by lordlala** [comment]
Hello! So, I am having this exact problem as well.

I installed it using pip as suggested. I use visual studio code as my editor. I have the latest python 3 installed.

The authentication works - I got the 2fa and I can even see in the variable that it says I'm authenticated.

In fact! I can even see my devices when I debug and look at the variables. So, why is it failing out with this NoneType error?
